Frends.AzureBlobStorage.DeleteContainer/Frends.AzureBlobStorage.DeleteContainer/DeleteContainer.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Frends.AzureBlobStorage.DeleteContainer.Definitions;
+using Azure.Storage.Blobs;
+using System.ComponentModel;
+
+namespace Frends.AzureBlobStorage.DeleteContainer
+{
+    public static class AzureBlobStorage
+    {
+        /// <summary>
+        /// Deletes a container from Azure blob storage.
+        /// [Documentation](https://tasks.frends.com/tasks#frends-tasks/Frends.AzureBlobStorage.DeleteContainer)
+        /// </summary>
+        /// <param name="input">Information about the container destination.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>Object { string Success }</returns>
+        public static async Task<Result> DeleteContainer([PropertyTab] Input input, CancellationToken cancellationToken)
+        {
+            if (input.ConnectionString == null || input.ContainerName == null)
+                throw new ArgumentNullException("Given parameter can't be empty.");
```

Comment:
Maybe split the connection string and container name checks to different if-statements and throw different errors depending on which parameter is empty, so it will be more clear for the user.